Fatal Error from FormTypeImplicit when using method call from variable

noofaq opened the pull request:
Solves #104

Nyholm commented on src/Visitor/Php/Symfony/FormTypeLabelImplicit.php:
```diff
@@ -32,6 +32,7 @@ public function enterNode(Node $node)
         $domain = null;
         // use add() function and look at first argument and if that's a string
         if ($node instanceof Node\Expr\MethodCall
+            && (!is_object($node->name) || method_exists($node->name, '__toString')) //fix for issue104
```
We do not need this comment. Rather give an example what scenario you are checking for.

noofaq commented:
@Nyholm I have removed the comment from the code completely as I am not sure it makes sense to add big comment in the middle of conditions. I guess it is not that hard to understand.

I have added new entry into changelog for 1.5.2 version as requested (feel free to correct it if needed)
